group_vars/all/release.yml

```diff
@@ -13,10 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-## Using this file
-#
-# This file is laid down only once and is used for overriding variables based on the file name.
-# This file will not be overwritten by upgrades.
+# The release tag to use for the repo and venvs
+# This can't be overriden because OSA is using group_vars.
+#openstack_release: "{{ rpc_release }}"
+
+# The release tag to use
+# Note that this is set here so that the openstack_release override in osa_defaults
+# is able to use it.
+rpc_release: "{{ lookup('pipe', 'cd /opt/rpc-openstack && git describe --tags --abbrev=0') }}"
```

Comment:
The overrides files are a user interface for implementing overrides and have always been empty (except for commented common overrides). We are not supposed to be editing them in any upgrades as they are supposed to be managed by the deployers.